<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pd_studio</class>
 <widget class="QMainWindow" name="pd_studio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>pd_studio</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(177, 203, 210);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget_menu">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>630</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="toolTipDuration">
         <number>0</number>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="tab_file">
         <property name="styleSheet">
          <string notr="true">border-color: rgb(64, 64, 64);</string>
         </property>
         <attribute name="title">
          <string>file</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_data_collect">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">border-color: rgb(64, 64, 64);
background-color: rgb(170, 255, 255);</string>
         </property>
         <attribute name="title">
          <string>data_collect</string>
         </attribute>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>671</width>
            <height>71</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>70</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(235, 255, 160);</string>
             </property>
             <property name="text">
              <string>Measure 1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>70</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(235, 255, 160);</string>
             </property>
             <property name="text">
              <string>Measure 2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>70</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(235, 255, 160);</string>
             </property>
             <property name="text">
              <string>Measure 3</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_4">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>70</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(235, 255, 160);</string>
             </property>
             <property name="text">
              <string>Measure 4</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_5">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>70</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(235, 255, 160);</string>
             </property>
             <property name="text">
              <string>Measure 5</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_6">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>70</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(235, 255, 160);</string>
             </property>
             <property name="text">
              <string>Measure 6</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_7">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>70</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(235, 255, 160);</string>
             </property>
             <property name="text">
              <string>Measure 7</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_8">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>70</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(235, 255, 160);</string>
             </property>
             <property name="text">
              <string>Measure 8</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="data_collect_horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_data_deal">
         <attribute name="title">
          <string>data_deal</string>
         </attribute>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>1</x>
            <y>6</y>
            <width>581</width>
            <height>76</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QGroupBox" name="gbox_data">
             <property name="minimumSize">
              <size>
               <width>45</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="title">
              <string/>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QPushButton" name="btn_import">
                  <property name="text">
                   <string>import</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btn_update">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="gbox_dataCut">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>44</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>240</width>
               <height>100</height>
              </size>
             </property>
             <property name="title">
              <string/>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <layout class="QGridLayout" name="glt_dataTrim_beginWith">
                <item row="1" column="0">
                 <widget class="QLineEdit" name="lineEdit_beginwith">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>20</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_beginwith">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>15</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Begin with:</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QGridLayout" name="glt_dataTrim_endWith">
                <item row="1" column="0">
                 <widget class="QLineEdit" name="lineEdit_endwith">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>20</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_endwith">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>15</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>End with:</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QPushButton" name="btn_trim_reset">
                  <property name="maximumSize">
                   <size>
                    <width>50</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Res</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btn_trim_save">
                  <property name="maximumSize">
                   <size>
                    <width>50</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Save</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QPushButton" name="btn_trim_export">
                  <property name="text">
                   <string>export</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="gbox_trans">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="title">
              <string/>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="QPushButton" name="btn_filter">
                  <property name="text">
                   <string>filter</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btn_filter_update">
                  <property name="text">
                   <string>update</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="vlt_freq">
                <item>
                 <widget class="QRadioButton" name="rbtn_fft">
                  <property name="text">
                   <string>fft</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="rbtn_wavelet">
                  <property name="text">
                   <string>wavelet</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="btn_fTrans_update">
                <property name="text">
                 <string>FUpdate</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="data_deal_horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget_main">
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>270</height>
         </size>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="tabsClosable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>tabWidget_main</sender>
   <signal>tabCloseRequested(int)</signal>
   <receiver>pd_studio</receiver>
   <slot>slot_closeTab()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>485</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>507</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_import</sender>
   <signal>clicked()</signal>
   <receiver>pd_studio</receiver>
   <slot>slot_fileOpen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_fileOpen()</slot>
  <slot>slot_showRightMenu()</slot>
  <slot>slot_closeTab()</slot>
  <slot>slot_fileSave()</slot>
 </slots>
</ui>
